src/types.ts

```diff
@@ -87,11 +87,11 @@ export interface TCPQueryResponse {
 }
 
 export interface BaseObject {
+  created?: number; // seconds
   id?: number;
+  modified?: number; // seconds
   tenantId?: number;
-  created?: number; // seconds
   updated?: number; // seconds
-  modified?: number; // seconds
 }
```

Comment:
If you are wondering what this change is about, I often select objects and alphabetise them because I get tired of looking up and down for keys I know exist 😅

Todo: add eslint rule that does it automatically.